_posts/2023-04-09-haberdashers-win-sr2023.md

```diff
@@ -0,0 +1,110 @@
+---
+title: Haberdashers' School wins Student Robotics Competition 2023!
+image: /images/content/blog/sr2022/sr2023-photo.JPG
+image_alt: All the teams from SR2023
+description: SR2023 came to an exciting conclusion with a victory for Haberdashers' School
+---
+
+{% include figure.html src="/images/content/blog/sr2023/sr2023-photo.JPG"
+           caption="All the teams from SR2023" %}
+
+That’s a wrap on Student Robotics 2023! [Haberdashers’ School](https://www.habsboys.org.uk/) came 1st amongst all of our competitors, after coming up against [Abingdon School](https://www.abingdon.org.uk/), [Gymnasium Markt Indersdorf](https://www.gym-indersdorf.de/), and [Harris Westminster Sixth Form](https://www.harriswestminstersixthform.org.uk/) in the finals. Our teams worked incredibly hard on their robots this year, and it showed in their performance in each round.
+
+## The game: Greed
+
+Greed required our team’s robots to capture tokens from their opponents and place them in their scoring zone. With Bronze, Silver, and Gold tokens, each worth a different number of game points, teams had to think carefully about their optimum strategy.
```

Comment:
Maybe emphasize the name of the game to clarify it's a name?
```suggestion
*Greed* required our team’s robots to capture tokens from their opponents and place them in their scoring zone. With Bronze, Silver, and Gold tokens, each worth a different number of game points, teams had to think carefully about their optimum strategy.
```
(maybe also in the preceding heading)